src/applications/mhv/medications/components/shared/FillRefillButton.jsx

```diff
@@ -45,12 +47,14 @@ const FillRefillButton = rx => {
             </p>
           </va-alert>
         </div>
-        <va-button
-          text={`Request ${isRefillable ? 'a refill' : 'the first fill'}`}
+        <button
```

Comment:
Just curious as to why change from the `<va-button>` va component to the `<button>`?

Reply:
I tried to add responsive styles to va-button.
But it didn't work. So, I changed it to button

Reply:
I see that makes sense. In the future you could try to leverage the `part` pseudo-element, it helps manipulate the shadow DOM styles, so something like this `va-button::part(button) { }.` . In this case since your not using the va component I would recommend to add the `type="button"` attribute to mimic the way the va set up their component, as well as getting rid of this linter issue.